### PR TITLE
 Format code according to pep8 style and add pep8 check in travis-ci

### PR DESCRIPTION
**Pull request by mzfr**
It is similar to [#117](https://github.com/mushorg/snare/pull/117) on snare

### CODE REVIEW

File: tanner/tests/test_api_server.py [hunk 1]
```diff
@@ -83,7 +83,8 @@ def get_app(self):
             "message": ["f387d46eaeb1454cadf0713a4a55be49", "e85ae767b0bb4b1f91b421b3a28082ef"]}}
         self.serv.api.return_sessions = mock_return_sessions
         request = await self.client.request("GET",
-                                            "/8fa6aa98-4283-4085-bfb9-a1cd3a9e56e4/sessions?filters=peer_ip:127.0.0.1 start_time:1497890400 user_agent:ngnix")
+                                            "/8fa6aa98-4283-4085-bfb9-a1cd3a9e56e4/sessions?\
+                                            filters=peer_ip:127.0.0.1 start_time:1497890400 user_agent:ngnix")
```
Comment (afeena) [inline tanner/tests/test_api_server.py]: Could you please check why this test if broken?
If you look at the travis log, you get:
```
FAIL: test_api_sessions_request (tanner.tests.test_api_server.TestAPIServer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/aiohttp/test_utils.py", line 418, in new_func
    func(self, *inner_args, **inner_kwargs))
  File "uvloop/loop.pyx", line 1364, in uvloop.loop.Loop.run_until_complete
  File "/opt/python/3.5.5/lib/python3.5/asyncio/futures.py", line 294, in result
    raise self._exception
  File "/opt/python/3.5.5/lib/python3.5/asyncio/tasks.py", line 240, in _step
    result = coro.send(None)
  File "/home/travis/build/mushorg/tanner/tanner/tests/test_api_server.py", line 88, in test_api_sessions_request
    assert request.status == 200
AssertionError: 
-------------------- >> begin captured logging << --------------------
aiohttp.server: ERROR: Error handling request
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/aiohttp/web_protocol.py", line 422, in start
    resp = yield from self._request_handler(request)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/aiohttp/web.py", line 306, in _handle
    resp = yield from handler(request)
  File "/home/travis/build/mushorg/tanner/tanner/api/server.py", line 62, in handle_sessions
    sessions = await self.api.return_sessions(applied_filters)
  File "/home/travis/build/mushorg/tanner/tanner/tests/test_api_server.py", line 77, in mock_return_sessions
    if type(filters) is dict and filters['peer_ip'] == "127.0.0.1" and \
KeyError: 'peer_ip'
aiohttp.access: INFO: 127.0.0.1 - - [25/Mar/2018:12:07:37 +0000] "GET /8fa6aa98-4283-4085-bfb9-a1cd3a9e56e4/sessions?
```

**Comment by mzfr** [comment]
@afeena I think this have something to do with #236. If you just look at the Travis log(in the end) it have old code for `test_rfi_emulator.py` for some reason.

But when I tried running test locally on my machine I am getting a lot more errors(12 to be precise) but none of the test is failing. 
* In `tanner.tests.test_api_server.TestAPIServer` I am getting 6 errors all saying `TypeError: get_app() takes 1 positional argument but 2 were given`
* In `tanner.tests.test_server.TestServer` I am getting 5 errors all saying `TypeError: get_app() takes 1 positional argument but 2 were given`
* In `tanner.tests.test_rfi_emulation.TestRfiEmulator` I am getting an error saying `TypeError: coroutines cannot be used with run_in_executor()`

Are you also getting similar type of error or am I doing something wrong here ?

Also I am not able to understand that [here](https://github.com/mushorg/tanner/blob/master/tanner/tests/test_rfi_emulation.py#L26) we are assigning some value to `data` but not using it. Can you explain why is it so ?

**Comment by afeena** [comment]
@mzfr Could you please rebase your branch?

If you noticed my comment, I said we can test in different ways. We can check that we call function with proper value and also can check that we get proper value after function call. In this case we can add one more test to check data. (Because in each test we should check exactly one case)

**Comment by xandfury** [comment]
On a side note, guys please use the develop branch for raising a PR. 
If not done so, sooner or later we would a pretty messy "merge conflict"(TM) on our hands :-)

File: tanner/tests/test_api_server.py (hunk 1 above)
Comment (afeena) [reply tanner/tests/test_api_server.py]: In this line when you put the part of request into new line, tabs also considered - that is why test is failed. Use "+" for line splitting instead

**Comment by mzfr** [comment]
@afeena I have added `noqa` to the line. Now we can keep it as it was and linter won't raise any issue.

File: tanner/tests/test_api_server.py [hunk 2]
```diff
@@ -83,8 +83,7 @@ def get_app(self):
             "message": ["f387d46eaeb1454cadf0713a4a55be49", "e85ae767b0bb4b1f91b421b3a28082ef"]}}
         self.serv.api.return_sessions = mock_return_sessions
         request = await self.client.request("GET",
-                                            "/8fa6aa98-4283-4085-bfb9-a1cd3a9e56e4/sessions?" +
-                                            "filters=peer_ip: 127.0.0.1 start_time: 1497890400 user_agent: ngnix")
+                                            "/8fa6aa98-4283-4085-bfb9-a1cd3a9e56e4/sessions?filters=peer_ip:127.0.0.1 start_time:1497890400 user_agent:ngnix")  # noqa
```
Comment (xandfury) [inline tanner/tests/test_api_server.py]: @mzfr Try this:
```
        request = await self.client.request("GET",
                                            "/8fa6aa98-4283-4085-"
                                            "bfb9-a1cd3a9e56e4/"
                                            "sessions?filters="
                                            "peer_ip:127.0.0.1"
                                            " start_time:1497890400"
                                            " user_agent:ngnix")
```

**Pull request merged**